packages/express-backend/ENDPOINTS.md

```diff
@@ -45,3 +45,14 @@ Gets a list of the items a particular user has posted.
 ### POST `/users/:userId/items`
 
 Creates a new listing from a user. The post body should contain an object meeting the requirements in the item schema. `title` and `description` are the only required fields.
+
+### GET ` /users/:userId/items-bought`
+
+Retrieves a list of items that a specific user has bought. Use the `userId` path variable to specify which user's purchased items you want to see. For example,
+`/users/abc/items-bought` to see items bought by the user with ID `abc`.
+
+### PATCH `/users/:userId/items/:itemId`
```

Comment:
`/users/:userId/items/:itemId` refers to an item that is owned by the item seller, so the `:userId` path variable should refer to the seller. Then the request body would contain a recipient id that would be passed to the `updateItemBuyerAndAddToUserBought` function.